test/lib/code_corps_web/views/conversation_view_test.exs

```diff
@@ -49,7 +94,25 @@ defmodule CodeCorpsWeb.ConversationViewTest do
       },
       "jsonapi" => %{
         "version" => "1.0"
-      }
+      },
+      "included" => [%{
+        "attributes" => %{
```

Comment:
I would probably shift the object down and right one to match the style used earlier.

Reply:
fixed